src/gui/DetailsWidget.ui

```diff
@@ -130,7 +130,7 @@
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
-      <number>0</number>
+      <number>4</number>
```

Comment:
Revert this to 0. QtCreator mixed this up

Reply:
Maybe it is better to activate the first tab in the constructor (to not depend on the form state)?

Reply:
I think the currentIndex is needed in the xml anyway, just set it to `0` so we don't add extra code for it